pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
This is probably left over from my initial kludgey implementation (which I did in about an hour when we needed it for BWM analyses of Fermi pulsars).  :-)